Ruins/src/main/java/atomicstryker/ruins/common/RuinTemplate.java

```diff
@@ -1248,4 +1310,34 @@ public RuinTemplateRule getVariant(final int selectorInitial, final int ruleInde
             }
         }
     }
+
+    public static class ClassException extends RuntimeException
+    {
+        private static final long serialVersionUID = -7630819126112904714L;
+
+        protected ClassException(String message)
+        {
+            super("RuinTemplate: " + message);

```

Comment:
Resolves to "RuinTemplate: template derp..."
Redundant information, and i would ask you to refactor those 3 exceptions into a single type anyway